prawns/crowdloans/src/processors/kusamaProcessor.ts

```diff
@@ -1,19 +1,18 @@
 import { SubstrateProcessor } from '@subsquid/substrate-processor';
 import crowdloanContributedHandler from '../handlers/crowdloan.contributed.event';
 import { SubstrateNetwork } from '../model';
+import { TypeormDatabase } from '@subsquid/typeorm-store'
+import { lookupArchive } from '@subsquid/archive-registry';
 
-const processor = new SubstrateProcessor('litentry_squid_crowdloans_kusama');
-
+const processor = new SubstrateProcessor(new TypeormDatabase());
 processor.setTypesBundle('kusama');
-processor.setBatchSize(500);
-processor.setIsolationLevel('REPEATABLE READ');
-processor.setBlockRange({ from: 7828000 });
```

Comment:
can we not keep the start block? I assume it still helps prevent wasted time scanning blocks before crowdloans started?

Reply:
Just seen the other processor, looks like this was omitted by accident